subiquity/common/types/__init__.py

```diff
@@ -86,10 +86,11 @@ class ApplicationState(enum.Enum):
     WAITING = enum.auto()
     RUNNING = enum.auto()
 
-    # States reported while unattended-upgrades is running.
-    # TODO: check if these should be dropped in favor of RUNNING.
+    # State reported while unattended-upgrades is running.
+    # TODO: check if this should be dropped in favor of RUNNING.
```

Comment:
They should only be distinct if a UI is going to care about the distinction and I think the way things are going the answer there is "no"?

Reply:
Currently, we have a distinction in the text UI so I'm tempted to leave it as is for now, wdyt?

Reply:
ok then!